packages/buckets/src/api/index.ts

```diff
@@ -617,17 +647,20 @@ export async function bucketsPushPath(
       client.send(req)
 
       if (source.content) {
-        const process = await block({ size: 32768, noPad: true })
-        for await (const chunk of process(source.content)) {
+        for await (const chunk of source.content) {
```

Comment:
Simple go through the chunked content... where chunks _might_ be extra big, or extra small.